.github/workflows/stale_bot.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-pr-label: 'stale'
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it has not had activity within 90 days.
+            It will be automatically closed if no further activity occurs within 30 days.
+          close-pr-message: >
+            This pull request has been automatically closed due to inactivity.
```

Comment:
The name mentions both PRs and issues, the messages/configuration in this file is only for PRs, though. Regarding the wording, it would be nice to guide the submitter in the message with what's expected to be done here as well. I believe in Ansible, their bot also asks a question like "What can you do to move this forward?" to encourage the submitter to take action.

Reply:
Good points @tyll - any suggestions for text?

Reply:
I add some minor changes, but feel free to make further suggestions, want this to be as contributor-friendly as possible while also keeping a realistic backlog for the maintainers :smile: